specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2021-03-01/managedClusters.json

```diff
@@ -65,6 +65,49 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.ContainerService/locations/{location}/os-options": {
```

Comment:
This violates resource type naming constraints.  Must be lowerCamelCase with only ascii alpha chars.  See https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#arguments-for-crud-on-resource

Reply:
Sure, change to osOptions.